checker/check-request-parameters-max-length-updated_test.go

```diff
@@ -0,0 +1,51 @@
+package checker_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tufin/oasdiff/checker"
+	"github.com/tufin/oasdiff/diff"
+)
+
+func TestRequestParameterMaxLengthIncreasedCheck(t *testing.T) {
+	s1, err := open("../data/checker/request_parameter_max_length_updated_base.yaml")
+	require.NoError(t, err)
+	s2, err := open("../data/checker/request_parameter_max_length_updated_revision.yaml")
+	require.NoError(t, err)
+
+	d, osm, err := diff.GetWithOperationsSourcesMap(getConfig(), s1, s2)
+	require.NoError(t, err)
+	errs := checker.CheckBackwardCompatibilityUntilLevel(singleCheckConfig(checker.RequestParameterMaxLengthUpdatedCheck), d, osm, checker.INFO)
+	require.Len(t, errs, 1)
+	require.Equal(t, checker.ApiChange{
+		Id:          "request-parameter-max-length-increased",
+		Text:        "for the 'query' request parameter 'category', the maxLength was increased from '10' to '15'",
```

Comment:
I recommend to change to:
"for the 'query' request parameter 'category', maxLength was increased from '10' to '15'",

(without "the")

Reply:
do we really want to change it when other checkers follow the same pattern? 

e.g. `request-parameter-max-length-decreased: for the %s request parameter %s, the maxLength was decreased from %s to %s`, I've no strong opinion but usually prefer to follow previous patterns or change them all, but changing them means potential issues on `err-ignore` functionality.